romancal/regtest/test_wfi_pipeline.py

```diff
@@ -547,3 +547,41 @@ def test_processing_pipeline_all_saturated(rtdata, ignore_asdf_paths):
     assert model.meta.cal_step.assign_wcs == "SKIPPED"
     assert model.meta.cal_step.flat_field == "SKIPPED"
     assert model.meta.cal_step.photom == "SKIPPED"
+
+
+@pytest.mark.bigdata
+def test_pipeline_suffix(rtdata, ignore_asdf_paths):
+    """Tests passing suffix to the pipeline"""
```

Comment:
```suggestion
    """
    Tests passing suffix to the pipeline
    
    Note that this test mimics how the pipeline is run in OPS.
    
    Any changes to this test should be coordinated with OPS.
    """
```
Do you think adding a note about this mimicking OPS runs is worthwhile (and not overly prescriptive)?